### ISSUE TITLE
Questions About Autoencoder Training: Loss Function, Training Steps, and Learning Rate Scheduler

### ISSUE THREAD

**Issue by SerdarHelli** [Tencent/Hunyuan3D-2#39]
Which loss function did you use when training the autoencoder? Did you directly compute the MSE between the predicted SDF values and the ground truth? Additionally, how many steps did you train for? Did you use a learning rate scheduler? If so, which one did you use?

These are questions I've always been curious about, as they're often not fully detailed in the paper.

**Comment by Zeqiang-Lai** [comment]
The loss function is elucidated in our report, it includes a reconstruction loss and kl loss. We use a linear warmup and constant lr. The vae is trained for roughly 300k steps.

**Comment by animemory** [comment]
> 300k

the batch size？
